packages/botonic-plugin-contentful/src/nlp/locales.ts

```diff
@@ -1,6 +1,12 @@
 import { stemmerFor } from './stemmer'
 
+// TODO convert to string enum?
 export type Locale = string
+export const SPANISH = 'es'
+export const CATALAN = 'ca'
+export const ENGLIGH = 'en'
+export const PORTUGUESE = 'pt'
+export const POLISH = 'pl'
```

Comment:
I'm not sure if enums exists as is in Javascript, but maybe you could just create the object languages with `freeze` in this way:
```
export const locales = Object.freeze({
SPANISH: 'es',
CATALAN:'ca',
....
})
```
when accessing them would look like `locales.SPANISH` which looks pretty good.

Reply:
in TS there's enum. in the future, I'll give it a try